### PR TITLE
fix(suite): Fix TruncateWithTooltip

### LINKED ISSUE #10905: Empower minor issues p 2
**Describe the bug**
6.`Close` button not properly aligned in `Update firmware complete` modal
7. Send and receive buttons are active even account not properly loaded, buttons disappear after you click them 
8. Account name is rendered as context help sometimes /probably due that it didnt fit to left panel before when ARS fiat value enabled/

**Info:**
 - Suite version: desktop 24.2.0 (906dcd476ef2e5919147b4f86e299b6e3b607d96)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/24.2.0 Chrome/118.0.5993.129 Electron/27.0.4 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T2B1 2.6.4 regular (revision 42e9ed0e09033d474dee1a560fe5870646fa440e)
 - Transport: BridgeTransport 2.0.33

**Screenshots:**
6.
image

7.
image

8.

https://github.com/trezor/trezor-suite/assets/31506317/2e73b3aa-a64a-4f13-87a3-94ff7884b0a8

https://github.com/trezor/trezor-suite/assets/31506317/0327cf57-0717-4784-aa01-7eec5c737e0c

### PR DESCRIPTION
**Pull request by jvaclavik**
## Related Issue

Resolve #10905

### CODE REVIEW

File: packages/components/src/components/typography/TruncateWithTooltip/TruncateWithTooltip.tsx [hunk 1]
```diff
@@ -17,13 +17,19 @@ export const TruncateWithTooltip = ({ children, delayShow }: TruncateWithTooltip
...
-
-            if (scrollWidth > clientWidth) {
+        if (!containerRef.current) return;
+        const { scrollWidth, scrollHeight } = containerRef.current;
+        const resizeObserver = new ResizeObserver(entries => {
+            const { inlineSize: width, blockSize: height } = entries[0].borderBoxSize?.[0];
+            if (scrollWidth > width || scrollHeight > height) {
                 setIsTooltipVisible(true);
+            } else {
+                setIsTooltipVisible(false);
```
Comment (peter-sanderson) [inline packages/components/src/components/typography/TruncateWithTooltip/TruncateWithTooltip.tsx]:
```suggestion
                setIsTooltipVisible(scrollWidth > width || scrollHeight > height);
```

And delete the  `if` ;)

**Review by peter-sanderson — APPROVED** [review]
just plese that one comment ;)

**Comment by komret** [comment]
@jvaclavik check unit testes please

**Pull request merged**